src/Tables/molecules/TypeTable.tsx

```diff
@@ -42,26 +45,75 @@ const HeaderItem = styled.div<{fixedWidth?: number, alignment?: TypeCellAlignmen
 
 `;
 
+const LoadingText = styled.div`
+  color: hsla(195, 10%, 52%, 0.72);
+}
```

Comment:
Stray closing tag? (unless I missed something)

Reply:
Thank you for mentioning it. I guess I did a bad cut.
I have updated.